mail_cleanup/models/fetchmail_server.py

```diff
@@ -99,25 +86,29 @@ def _purge_fetchmail_server(self, server, imap_server):
                 # Delete message
                 imap_server.store(num, '+FLAGS', '\\Deleted')
             except Exception:
-                _logger.exception('Failed to remove mail from %s server %s.',
-                                  server.type, server.name)
+                _logger.exception(
+                    'Failed to remove mail from %s server %s.',
+                    server.type, server.name)
                 failed += 1
             count += 1
-        _logger.info("Removed %d email(s) on %s server %s; "
-                     "%d succeeded, %d failed.", count, server.type,
-                     server.name, (count - failed), failed)
+        _logger.info(
+            'Removed %d email(s) on %s server %s;'
+            ' %d succeeded, %d failed.', count, server.type,
+            server.name, (count - failed), failed)
 
     @api.multi
     def fetch_mail(self):
-        """ Called before the fetch, in order to clean up
-             right before retrieving emails. """
+        # Called before the fetch, in order to clean up right before
+        # retrieving emails.
         context = self.env.context.copy()
         context['fetchmail_cron_running'] = True
         for server in self:
             _logger.info('start cleaning up emails on %s server %s',
                          server.type, server.name)
-            context.update({'fetchmail_server_id': server.id,
-                            'server_type': server.type})
+            context.update({
```

Comment:
Are all these context update required ? It seems it's never used...

Reply:
I've found a number of hits on an instance w/ multiple repos in scope, some of them are reading this key from a context, so I'm not sure if it is useless

![image](https://user-images.githubusercontent.com/34241889/47570772-a63d4380-d93f-11e8-94cf-975a5afa8a08.png)

regarding `server_type` - I've found only two hit that set it, but nothing was reading that key